### ISSUE TITLE
ru_ru.lang

### ISSUE THREAD

**Issue by aigiz010** [LordDeatHunter/Alloy-Forgery#35]
[ru_ru.zip](https://github.com/LordDeatHunter/Alloy-Forgery/files/7964364/ru_ru.zip)

**Comment by MJRamon** [comment]
Fixed. Pproper Russian language terms are used.

```
{
  "block.alloy_forgery.bricks_forge_controller": "Регулятор литейной печи из кирпичей",
  "block.alloy_forgery.stone_bricks_forge_controller": "Регулятор литейной печи из каменных кирпичей",
  "block.alloy_forgery.deepslate_bricks_forge_controller": "Регулятор литейной печи из глубинносланцевых кирпичей",
  "block.alloy_forgery.end_stone_bricks_forge_controller": "Регулятор литейной печи из эндерняковых кирпичей",
  "block.alloy_forgery.polished_blackstone_forge_controller": "Регулятор литейной печи из полированного чернита",

  "tooltip.alloy_forgery.forge_tier": "Уровень: %d",
  "tooltip.alloy_forgery.fuel_capacity": "Ёмкость топлива: %d",

  "container.alloy_forgery.forge_controller": "Литейная печь",

  "container.alloy_forgery.rei.title": "Литейная печь",
  "container.alloy_forgery.rei.min_tier": "Требуемый уровень: %d",
  "container.alloy_forgery.rei.fuel_per_tick": "Топливо за тик: %d",

  "message.alloy_forgery.invalid_multiblock": "Мультиблочная структура построена не правильно",
  "itemGroup.alloy_forgery.alloy_forgery": "Ковка сплава"
}
``

**Comment by Noaaan** [comment]
Fixed by #50
